### DISCUSSION TITLE
PR Title Checker / check (pull_request_target)

### DISCUSSION THREAD

**Discussion by ajay-dhangar** [priyankarpal/projectshut#1178] [General]
Can you help me how to solve this?

[![image](https://github.com/priyankarpal/ProjectsHut/assets/99037494/64f8ef42-72ab-4aa5-8e94-32bd19f5b64a)](https://github.com/priyankarpal/ProjectsHut/pull/1177)

**Comment by priyankarpal** [comment]
add a good title suppose yu added a new feature then the title would be `feat: describe your message(short)`
